seed/static/seed/partials/geocode_modal.html

```diff
@@ -85,10 +86,16 @@ <h4 class="modal-title" id="exportModalLabel" ng-show="geocode_state === 'result
             </div>
         </div>
     </div>
+    <div ng-show="geocode_state === 'fail'">
+        <div class="alert alert-danger"  ng-if="error_message">
+            <p>{$ error_message | translate $}</p>
+        </div>
+    </div>
 
 </div>
 <div class="modal-footer">
     <button type="button" class="btn btn-primary" ng-click="geocode_buildings()" ng-show="geocode_state === 'verify' && has_api_key && has_enough_geocoding_columns" translate>Geocode</button>
     <button type="button" class="btn btn-default" ng-show="geocode_state == 'verify'" ng-click="cancel()" translate>Cancel</button>
     <button type="button" class="btn btn-default" ng-show="geocode_state === 'result'" ng-click="close()" translate>Close</button>
+    <button type="button" class="btn btn-default" ng-show="geocode_state === 'fail'" ng-click="close()" translate>Close</button>
```

Comment:
This works but we could probably DRY things up a bit with this.
```suggestion
    <button type="button" class="btn btn-default" ng-show="geocode_state === 'fail' || geocode_state === 'result'" ng-click="close()" translate>Close</button>
```

Reply:
thank you!